assets/src/scss/pages/_page.scss

```diff
@@ -121,3 +121,62 @@
     outline: 0 none;
   }
 }
+
+.bottom-navigation {
+  height: 70px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-weight: var(--font-weight-bold);
+  font-family: var(--font-family-heading);
+
+  a:hover {
+    text-decoration: none;
+  }
+
+  &-next {
+    margin-inline-start: auto;
+
+    &::after {
+      content: "\2192";
+      font-weight: var(--font-weight-bold);
+      margin-left: $sp-1;
+    }
+  }
+
+  &-prev {
+    &::before {
+      content: "\2190";
+      font-weight: var(--font-weight-bold);
+      margin-inline-end: $sp-1;
+    }
+  }
+}
+
+html[dir="rtl"] {
+  .bottom-navigation {
+    &-next {
+      margin-inline-start: 0;
+      margin-inline-end: auto;
+
+      &::after {
+        display: inline-block;
+        transform: scaleX(-1);
+        margin-inline-end: $sp-1;
```

Comment:
This should no longer be necessary 🤔 That's the whole point of using `margin-inline-...`

Reply:
The arrows are one direction only, need to flip them when the site changes from LTR to RTL

Reply:
Yes I understand that, but that's the transform, the margin should remain the same if you use `margin-inline-end` and `margin-inline-start`

Reply:
Btw instead of `scaleX` maybe we should use `rotate`? Seems more clear to me.